store/src/main/java/com/dropbox/android/external/store4/Store.kt

```diff
@@ -6,13 +6,11 @@ import kotlinx.coroutines.flow.filterNot
 import kotlinx.coroutines.flow.first
 import kotlinx.coroutines.flow.transform
 
-// possible replacement for [Store] as an internal only representation
-// if this class becomes public, should probaly be named IntermediateStore to distingush from
-// Store and also clarify that it still needs to be built/open? (how do we ensure?)
 interface Store<Key, Output> {
```

Comment:
would be good to add some top level documentation to the store type. a lot of time people will just read the doc for the type in the IDE

Reply:
done